static/app/components/sidebar/index.spec.tsx

```diff
@@ -49,12 +49,12 @@ describe('Sidebar', function () {
   });
 
   it('renders', async function () {
-    renderSidebar();
+    renderSidebar({organization});
     expect(await screen.findByTestId('sidebar-dropdown')).toBeInTheDocument();
   });
 
   it('renders without org', async function () {
-    renderSidebar({organization: undefined});
+    renderSidebar({organization: null});
```

Comment:
Had to make some slight adjustments to our custom `render` RTL function to allow organization to not just default to `OrganizationFixture`